deploy/000_register_hub.ts

```diff
@@ -28,7 +29,7 @@ const deployFunction: DeployFunction = async function (hre: HardhatRuntimeEnviro
     const chainlinkDeployment = await hre.deployments.deploy('Oracle', {
       from: deployer,
       contract: { abi: CHAINLINK_ABI, bytecode: CHAINLINK_BYTECODE },
-      args: [WETH, CHAINLINK_REGISTRY, governor],
+      args: [WETH, CHAINLINK_REGISTRY, BigNumber.from(2).pow(32).sub(1), governor],
```

Comment:
also, this is due to a dca-v2-core change, right ?

Reply:
Indeed